src/mango/test/README.md

```diff
@@ -22,8 +22,13 @@ Test configuration
 
 The following environment variables can be used to configure the test fixtures:
 
- * `COUCH_HOST` - root url (including port) of the CouchDB instance to run the tests against. Default is `"http://127.0.0.1:15984"`.
- * `COUCH_USER` - CouchDB username (with admin premissions). Default is `"testuser"`.
- * `COUCH_PASSWORD` -  CouchDB password. Default is `"testpass"`.
- * `COUCH_AUTH_HEADER` - Optional Authorization header value. If specified, this is used instead of basic authentication with the username/password variables above.
- * `MANGO_TEXT_INDEXES` - Set to `"1"` to run the tests only applicable to text indexes.
+ * `COUCH_HOST` - root url (including port) of the CouchDB instance to
```

Comment:
Eyeballing this I'm not seeing anything other that reformatting but I'm not 100%. A good example of why its not a good idea to mix style and logic changes. Unless there's a change I'm not seeing please remove this hunk.